Mathlib/CategoryTheory/Monoidal/Functorial.lean

```diff
@@ -57,10 +59,12 @@ class LaxMonoidalStruct (F : C → D) [Functorial.{v₁, v₂} F] where
 -- but that isn't the immediate plan.
 /-- An unbundled description of lax monoidal functors. -/
 class LaxMonoidal (F : C → D) [Functorial.{v₁, v₂} F] extends LaxMonoidalStruct F where
+  /-- naturality of the tensorator -/
```

Comment:
https://github.com/leanprover/std4/pull/269 will make it so you don't have to add all these